### DISCUSSION TITLE
Add Search for Pal Selection

### DISCUSSION THREAD

**Discussion by Biubush** [KrisCris/Palworld-Pal-Editor#21] [Ideas]
**Is your feature request related to a problem? Please describe.**
更改帕鲁种类时下拉框找帕鲁的方法实在不太友好，应该是按帕鲁序号排序的，我觉得可以按名称排序，最人性化设计应该是搜索框快速匹配选择

**Describe the solution you'd like**
![image](https://github.com/KrisCris/Palworld-Pal-Editor/assets/15962307/d63fd6b0-0727-4c81-9aca-c31598354079)
在下拉框处添加输入框快速匹配帕鲁

**Describe alternatives you've considered**
无

**Additional context**
无

**Comment by KrisCris** [comment]
Thanks for your suggestion. Currently the list is sorted by PalDeck ID, with which I see no issue. 

Searching / Filtering will probably be supported in the future.
